.storybook/preview.js

```diff
@@ -53,9 +53,9 @@ export const parameters = {
 				"Guides",
 				["Contributing", "*", "Adobe Code of Conduct", "Changelog"],
 				"Components",
-				["*", ["Docs", "Default", "*"]],
+				["*", ["Docs", "Default", "Standard", "Sizing", "*"]],
```

Comment:
In this listing, how does Standard differ from Default? Those seems like synonyms to me.